### PR TITLE
Fix responsiveness and font size  of sponsors. Closes #136

### LINKED ISSUE #136: Responsiveness and font size in sponsors
1) Sponsors section needs to fit small screens a bit better 

sponsors-mobile

---

2) Font is larger for sponsors in past episodes than in future episodes/homepage for some reason 

Sponsors in past episodes
sponsors-previousepisodes

Sponsors in homepage
sponsors-homepage

### PR DESCRIPTION
**Pull request by housseindjirdeh**
## Mobile

sponsors-mobile
## Tablet

sponsors-tablet
## Laptop

sponsors-laptop

### CODE REVIEW

File: src/pages/episode/sections/show-notes/index.js [hunk 1]
```diff
@@ -16,9 +16,9 @@ function ShowNotes({episode}) {
     return <noscript />
   }
   return (
-    <section id="show-notes">
```
Comment (kentcdodds) [inline src/pages/episode/sections/show-notes/index.js:19]: The ID should stay so we can hard-link to the show notes section of the page :+1: (for example: https://javascriptair.com/episodes/2016-06-15/#show-notes)

**Comment by kentcdodds** [comment]
Looks good. Any chance we could bump the font size for desktop? It just looks kinda small. Maybe something like `30px`?

File: src/pages/episode/sections/show-notes/index.js (hunk 1 above)
Comment (housseindjirdeh) [inline src/pages/episode/sections/show-notes/index.js:19]: Oops you're right, put it back

**Comment by housseindjirdeh** [comment]
Done. `30px` is quite large though, `23px` looks pretty good :+1: 

sponsors-laptop

**Comment by kentcdodds** [comment]
Great! Thanks!

**Pull request merged**